src/stirling/source_connectors/socket_tracer/protocols/cql/types.h

```diff
@@ -103,6 +103,7 @@ struct Frame : public FrameBase {
   FrameHeader hdr;
   std::string msg;
   bool consumed = false;
+  bool discarded = false;
```

Comment:
@benkilimnik this seems like a remnant from https://github.com/pixie-io/pixie/pull/1715

Reply:
Removed in most recent commit